UI/Components/PossibleTimeSaveSettings.cs

```diff
@@ -111,6 +115,11 @@ void cmbGradientType_SelectedIndexChanged(object sender, EventArgs e)
             GradientString = cmbGradientType.SelectedItem.ToString();
         }
 
+        void chkDropDecimals_CheckedChanged(object sender, EventArgs e)
+        {
+            DropDecimals = !DropDecimals;
+        }
```

Comment:
Are you sure that we need this here? The data bindings might handle this automatically, but please test that to make sure.

Reply:
Oh, I guess we don't need it. I added it to make it more like the seconds/tenths/hundredths radio buttons, but I didn't realize those didn't have data bindings. (Tested to make sure)